tidb/datadog_checks/tidb/check.py

```diff
@@ -0,0 +1,74 @@
+from datadog_checks.base import OpenMetricsBaseCheck
+
+from .metrics import GO_RUNTIME_METRICS, PD_METRICS, TIDB_METRICS, TIKV_METRICS
+
+
+class TiDBCheck(OpenMetricsBaseCheck):
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances=None):
+        default_metrics = dict(TIDB_METRICS)
+        default_metrics.update(PD_METRICS)
+        default_metrics.update(TIKV_METRICS)
+        default_metrics.update(GO_RUNTIME_METRICS)
+
+        tidb_metrics = dict(TIDB_METRICS)
+        tidb_metrics.update(GO_RUNTIME_METRICS)
+
+        pd_metrics = dict(PD_METRICS)
+
+        tikv_metrics = dict(TIKV_METRICS)
+
+        default_instances = {
+            'pd': {
+                'prometheus_url': 'http://localhost:2379/metrics',
```

Comment:
`localhost` should be something configurable.
See my comment in the `spec.yaml` so it's possible to do something like:
```suggestion
                'prometheus_url': instances[0]['base_url'] + ':2379/metrics',
```